src/components/ProfilePage/AddressBox.tsx

```diff
@@ -0,0 +1,84 @@
+import { Box, Typography, alpha, useTheme } from '@mui/material';
+import OpenInNewIcon from '@mui/icons-material/OpenInNew';
+import ContentCopyIcon from '@mui/icons-material/ContentCopy';
+import { useMenuStore } from 'src/stores';
+import { useTranslation } from 'react-i18next';
+import {
+  AddressDisplayBox,
+  BackgroundBox,
+  PassImageBox,
+  ProfileIconButton,
+} from './AddressBox.style';
+import { ProfilePageTypography } from './ProfilePage.style';
+import { useEnsName } from 'wagmi';
+import { mainnet } from 'wagmi/chains';
+
+interface AddressBoxProps {
+  address?: string | undefined;
```

Comment:
Question mark is already says that it can be undefined, not need for additional `| undefined` part